### PR TITLE
added headers support

### PR DESCRIPTION
**Pull request by Dnile**
added headers support in the blackbox_exporter config yml,
Headers located under HTTPProbe in http.go.

updated readme accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ modules:
     timeout: 5s
     http:
       valid_status_codes: []  # Defaults to 2xx
+      headers:
+        header_title: header_content
```
Comment (brian-brazil) [inline README.md]: s/title/name/

File: http.go [hunk 2]
```diff
@@ -83,6 +83,12 @@ func probeHTTP(target string, w http.ResponseWriter, module Module) (success boo
 		return
 	}
 
+	if len(config.Headers) > 0 {
```
Comment (brian-brazil) [inline http.go]: You can just range over it without the if

**Comment by brian-brazil** [comment]
Can you add a unittest please?

**Comment by Dnile** [comment]
thanks for the comments,i'll add the unit test today.

**Comment by zerthimon** [comment]
+1

**Comment by brian-brazil** [comment]
Can you squash your commits and fix the conflict?

**Comment by brian-brazil** [comment]
Header support was added in another PR.

**Pull request closed without merging**